pytorch_lightning/core/step_result.py

```diff
@@ -134,6 +134,9 @@ def log(
         # sync across workers when using distributed training
         sync_fn = sync_fn or sync_ddp_if_available
         if sync_dist and isinstance(value, (torch.Tensor, numbers.Number)):
+            is_dist_initialized = torch.distributed.is_available() and torch.distributed.is_initialized()
```

Comment:
is here the available needed?

Reply:
Yes.

Reply:
The synchronisation won't happen if `is_dist_initialized` is False, and therefore there is no need to clone the variable.